src/components/pages/AdminHome/AdminCourses.js

```diff
@@ -1,11 +1,78 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
 import AdminSidebar from './AdminSidebar';
-import '../../../styles/index.less';
+import { connect } from 'react-redux';
+import CourseCard from './AdminCourseCard';
+import { useDispatch } from 'react-redux';
+import { getCourses } from '../../../redux/actions/coursesActions';
+import { useOktaAuth } from '@okta/okta-react';
+import { Layout, Input } from 'antd';
+const { Search } = Input;
```

Comment:
Let's remove `Search` since we're not using it.

Reply:
being told live in PR Critiques that this will be added eventually, can we get a codeComment that explains this intent?

Reply:
Tagging @macrowell70 for exposure.

Reply:
I have removed this import, there will be a need to import the custom search bar at a later date, but I don't think we need the placeholder for the import. Also removed 'Input' which was only imported for 'Search'